web/src/components/admin/connectors/Field.tsx

```diff
@@ -100,8 +103,11 @@ export function TextFormField({
   label,
   subtext,
   placeholder,
+  value,
```

Comment:
Might be worthwhile to consider consolidating all these parameters into a config object or create a subclass for a more highly customized version of this component.